src/nxdoc/web-ui/web-ui-tutorials/how-to-create-bulk-edit-form-studio.md

```diff
@@ -0,0 +1,89 @@
+---
+title: 'HOWTO: Create a Bulk Edit Form Using Nuxeo Studio'
+review:
+    comment: ''
+    date: '2021-12-21'
+    status: ok
+toc: true
+details:
+    howto:
+        excerpt: Learn how to create a bulk edit form using Nuxeo Studio Designer
+        level: Intermediate
+        tool: null
+        topics: 'Web UI, Studio Designer, UIElements, Bulk'
+labels:
+    - tutorial
+tree_item_index: 497
+---
+
+{{! excerpt}}
+In this tutorial you will learn how to create a form to edit metadata on a large set of documents.
+{{! /excerpt}}
+
+## Requirements
+
+{{#> callout type='info'}}
+This capability is available since LTS 2021 with Web UI 3.0.9.
+{{/callout}}
+
+- A [Contract document type]({{page version='' space='nxdoc' page='web-ui-document-layouts'}}#create-a-contract-document-type) created in Nuxeo Modeler.
+- The Nuxeo [Web UI addon](https://connect.nuxeo.com/nuxeo/site/marketplace/package/nuxeo-web-ui) installed on your instance.
+- In Studio Modeler > **Settings** > **Application Definition**, make sure that **Nuxeo Web UI** is in the **Packages to Install** list.
+- Activate the `Select All and Bulk Actions` feature by adding the `nuxeo.selection.selectAllEnabled=true` property in your [nuxeo.conf]({{page page="configuration-parameters-index-nuxeoconf"}}) file.
+
+## Create the Structure
+
+- In Studio Designer, click on the **RESOURCES** tab.
+- Select the `UI` folder and click on the `create` button at the bottom of the screen.
+- Create a new folder and name it `bulk`.
+
+## Create a Bulk Edit Layout
+
+To make things faster, we will reuse the same form as the one used for the edit layout of the Contract document type.
+
+- From the **RESOURCES** tab, open the **document** > **contract** folder and open the `nuxeo-contract-edit-layout.html` file.
+- Copy the content of the layout (Ctrl/Cmd + A then Ctrl/Cmd + C).
+- Select the `bulk` folder you created previously and click on the `create` button at the bottom of the screen.
+- Create a new `empty file` and name it `nuxeo-bulk-edit-contracts-layout`.
```

Comment:
Shouldn't the file name end with `html`, like `nuxeo-bulk-edit-contracts-layout.html`?

Reply:
Indeed, good catch! Fixed that, thanks Ricardo!